notes/2019/2019-11-07.md

```diff
@@ -0,0 +1,51 @@
+# 2019-November-07 ESLint TSC Meeting Notes
+
+## Transcript
+
+https://gitter.im/eslint/tsc-meetings/archives/2019/11/07
+
+## Attending
+
+* Brandon Mills (@btmills) - TSC
+* Toru Nagashima (@mysticatea) - TSC
+* Kevin Partington (@platinumazure) - TSC
+* Ilya Volodin (@ilyavolodin) - TSC
+
+Additionally, the following individuals are not in attendance but support consensus:
+
+* Nicholas C. Zakas (@nzakas) - TSC
+* Gyandeep Singh (@gyandeeps) - TSC
+
+## Topics
+
+### [New: `ESLint` Class Replacing `CLIEngine`](https://github.com/eslint/rfcs/pull/40)
+
+* @mysticatea hopes to advance this RFC to the Final Commenting stage after this meeting.
+* Per @ilyavolodin, "This is a huge change to the core, so we need to get as much feedback as possible before we decide to go one way or the other."
+* @mysticatea considered using Node.js streams instead of async iterables, but that would require minimum Node.js `v10.17.0`, whereas RFC44 decided on a minimum version of `10.0.0`.
+* Use of async iterables will not preclude adding stream-based APIs in the future.
+* Rather than adding async linting and async plugin loading and deprecating some `CLIEngine` APIs in a single change, an alternative would be an `executeOnFilesAsync` method to support parallel linting and formatters reporting progress.
+* @platinumazure prefers to do everything as a single breaking change to minimize churn for users and maintenance cost compared to a sequence of major API changes.
+* @ilyavolodin wants to enable parallel linting but doesn't see that completely changing ESLint APIs to async buys us enough to justify the effort, especially since the CLI is mostly async.
```

Comment:
```suggestion
* @ilyavolodin wants to enable parallel linting but doesn't see that completely changing ESLint APIs to async buys us enough to justify the effort, especially since the CLI is mostly sync.
```

Reply:
Ha, that's an important distinction - thanks for catching it!